site/docs/reference/Connectors/materialization-connectors/BigQuery.md

```diff
@@ -26,16 +26,6 @@ To use this connector, you'll need:
 
     See [Setup](#setup) for detailed steps to set up your service account.
 
-The Flow collections you materialize must accommodate the following naming restrictions:
```

Comment:
The connector handles these cases now automatically without erroring, so these docs aren't applicable anymore